internal/dbmodel/group_test.go

```diff
@@ -36,3 +36,20 @@ func TestGroupEntry(t *testing.T) {
 	c.Assert(result.Error, qt.IsNil)
 	c.Assert(ge3, qt.DeepEquals, ge)
 }
+
+// test we hard delete groups
+func TestHardDeleteGroupEntry(t *testing.T) {
```

Comment:
i think this test would've passed with soft deletes as well.. what you need to do is
1. create a group
2. delete the created group
3. try to create another group with the same name

Reply:
did it